### DISCUSSION TITLE
Really nice exp

### DISCUSSION THREAD

**Discussion by rhjensen79** [adhocteam/pushup#111] [General]
Hi. 

Just want to say really nice experience, getting started with pushup. 

Been looking for a nice Golang framework for web, for quite some time.

Keep up to the good work :-)

**Comment by mickgardner** [comment]
+1
It's been a long time since I was genuinely excited by a new web framework of any sort and Pushup genuinely excites me with its functionality and simplicity. I really look forward to seeing where this framework goes in the future. Thank you.
